tests/features/payment-gateway-suggestions/payment-gateway-suggestions.php

```diff
@@ -64,13 +64,13 @@ public function get_mock_specs() {
 	 */
 	public function test_get_default_specs() {
 		add_filter(
-			'woocommerce_admin_payment_gateway_suggestions_data_sources',
+			DataSourcePoller::FILTER_NAME,
```

Comment:
It appears you are missing a import here for DataSourcePoller:
`use Automattic\WooCommerce\Admin\Features\PaymentGatewaySuggestions\DataSourcePoller;`